R/utils.R

```diff
@@ -0,0 +1,54 @@
+
+escape_vars <- function(names) {
+  if (length(names) == 0)
+    return(names)
+  names <- sapply(names, function(name) {
```

Comment:
can you please change `sapply` to `vapply`? I had too many problems with the former in `MPTinR` as it does not guarantee a specific return value. So I only use `sapply` in interactive use, but do not want it in package code.